web/src/main/webapp/WEB-INF/jsp/managestudy/viewSectionDataPreview.jsp

```diff
@@ -15,318 +15,322 @@
- <td class="table_cell_noborder" ><c:out value="${studySubject.label}"/><br>
- </td>
- <c:choose>
- <c:when test="${study.studyParameterConfig.personIdShownOnCRF == 'true'}">
-
- <td class="table_cell_top" ><b><fmt:message key="person_ID" bundle="${resword}"/>:</b><br></td>
- <td class="table_cell_noborder" ><c:out value="${subject.uniqueIdentifier}"/><br></td>
- </c:when>
- <c:otherwise>
- <td class="table_cell_top" ><b><fmt:message key="person_ID" bundle="${resword}"/>:</b><br></td>
- <td class="table_cell_noborder" ><fmt:message key="na" bundle="${resword}"/></td>
- </c:otherwise>
- </c:choose>
-</tr>
-<tr>
- <td class="table_cell_noborder"><b><fmt:message key="study_site" bundle="${resword}"/>:</b><br></td>
- <td class="table_cell_noborder" ><c:out value="${studyTitle}"/><br></td>
- <td class="table_cell_top" ><b><fmt:message key="age" bundle="${resword}"/>:</b><br></td>
- <td class="table_cell_noborder"><c:choose><c:when test="${age!=''}"><c:out value="${age}"/></c:when>
-	 <c:otherwise> <fmt:message key="na" bundle="${resword}"/></c:otherwise></c:choose><br></td>
-</tr>
-<tr>
- <td class="table_cell_noborder" ><b><fmt:message key="event" bundle="${resword}"/>:</b></td>
- <td class="table_cell_noborder" ><c:out value="${studyEvent.studyEventDefinition.name}"/> (<fmt:formatDate value="${studyEvent.dateStarted}" pattern="${dteFormat}"/>)</td>
- <td class="table_cell_top" ><b><fmt:message key="date_of_birth" bundle="${resword}"/>:</b><br></td>
- <td class="table_cell_noborder" ><fmt:formatDate value="${subject.dateOfBirth}" pattern="${dteFormat}"/><br></td>
-</tr>
-<tr>
-  <td class="table_cell_noborder" ><b><fmt:message key="interviewer" bundle="${resword}"/>:</b></td>
-  <td class="table_cell_noborder" ><c:out value="${EventCRFBean.interviewerName}"/> (<fmt:formatDate value="${EventCRFBean.dateInterviewed}" pattern="${dteFormat}"/>)</td>
-  <td class="table_cell_top" ><b><fmt:message key="Gender" bundle="${resword}"/>:</b></td>
-  <td class="table_cell_noborder" ><c:choose>
-    <c:when test="${subject.gender==109}"><fmt:message key="M" bundle="${resword}"/></c:when>
-    <c:when test="${subject.gender==102}"><fmt:message key="F" bundle="${resword}"/></c:when>
-    <c:otherwise><c:out value="${subject.gender}"/></c:otherwise>
-</c:choose></td>
-</tr>
-</table>
-
-
-
-<span ID="spanAlert-interviewDate" class="alert"></span>
-</td>
-				</tr>
-			</table>
-
-	  	</td>
-	</tr>
-				</table>
-
-		</div>
-
-	</div></div></div></div></div></div></div>
+    <table border="0" cellpadding="0" cellspacing="0">
+        <tr id="CRF_infobox_closed" style="display: none;">
+            <td style="padding-top: 3px; padding-left: 6px; width: 90px;" nowrap>
+                <a href="javascript:leftnavExpand('CRF_infobox_closed'); leftnavExpand('CRF_infobox_open');"><img src="images/sidebar_expand.gif" border="0" align="right" hspace="10"></a>
+
+                <b>CRF Info</b>

```

Comment:
If you are removing hardcoded strings in this file, this one should also be added to a properties file.